cppcache/include/geode/CacheStatistics.hpp

```diff
@@ -97,11 +97,11 @@ class APACHE_GEODE_EXPORT CacheStatistics {
   virtual time_point getLastAccessedTime() const;
 
  private:
-  virtual void setLastAccessedTime(time_point lat);
-  virtual void setLastModifiedTime(time_point lmt);
+  virtual void setLastAccessedTime(const time_point& tp);
+  virtual void setLastModifiedTime(const time_point& tp);
 
-  std::atomic<time_point::duration::rep> m_lastAccessTime;
-  std::atomic<time_point::duration::rep> m_lastModifiedTime;
+  std::atomic<time_point::duration::rep> last_accessed_{0};
```

Comment:
This change also inlines into client code. Don't get me wrong, the original constructor was just as wrong, it should have just been moved into the source file. Save this sort of thing for internal implementation, not published headers.